backend/benefit/applications/models.py

```diff
@@ -260,3 +281,71 @@ class Meta:
         db_table = "bf_applications_applicationbasis"
         verbose_name = _("application basis")
         verbose_name_plural = _("application bases")
+
+
+class Employee(UUIDModel, TimeStampedModel):
+    application = models.OneToOneField(
+        Application,
+        verbose_name=_("application"),
+        related_name="employee",
+        on_delete=models.CASCADE,
+    )
+
+    first_name = models.CharField(max_length=128, verbose_name=_("first name"))
+    last_name = models.CharField(max_length=128, verbose_name=_("last name"))
+    social_security_number = models.CharField(
+        max_length=11, verbose_name=_("social security number")
+    )
+    phone_number = models.CharField(max_length=64, verbose_name=_("phone number"))
+    email = models.EmailField(blank=True, verbose_name=_("email"))
+
+    employee_language = models.CharField(
+        choices=APPLICATION_LANGUAGE_CHOICES,
+        default=APPLICATION_LANGUAGE_CHOICES[0][0],
+        max_length=2,
+    )
+
+    job_title = models.CharField(
+        blank=True, verbose_name=_("job title"), max_length=128
+    )
+    monthly_pay = models.DecimalField(
+        verbose_name=_("monthly pay"),
+        decimal_places=2,
+        max_digits=7,
+        blank=True,
+        null=True,
+    )
+    vacation_money = models.DecimalField(
+        verbose_name=_("vacation money"),
+        decimal_places=2,
+        max_digits=7,
+        blank=True,
+        null=True,
+    )
+
+    other_expenses = models.DecimalField(
+        verbose_name=_("other expenses"),
+        decimal_places=2,
+        max_digits=7,
+        blank=True,
+        null=True,
+    )
+    working_hours = models.DecimalField(
+        verbose_name=_("working hour"),
+        decimal_places=1,
+        max_digits=4,
+        blank=True,
+        null=True,
+    )
+
+    # TODO: Adding missing file fields or a separate Attachment relationship
+    # power_of_attorney = models.FileField()
+    # collective_bargaining_agreement = models.FieldField()
```

Comment:
I think that the collective_bargaining_agreement would be a freeform text input (not the actual file itself)

Reply:
Ah, right, yeah I thought it was part of the attachments in the third step